packages/ramp-core/src/fixtures/legend/components/legend-options.vue

```diff
@@ -120,84 +120,90 @@
-     */
-    toggleSymbology(): void {
-        if (this.legendItem._controlAvailable(Controls.Symbology)) {
-            this.legendItem.displaySymbology = !this.legendItem.displaySymbology;
-        }
-    }
+export default defineComponent({
+    name: 'LegendOptionsV',
+    props: ['legendItem'],
+    data() {
+        return {
+            removeLayerEntry: call(LegendStore.removeLayerEntry)
+        };
+    },
+    methods: {
+        /**
+         * Display symbology stack for the layer.
+         */
+        toggleSymbology(): void {
+            if (this.legendItem._controlAvailable(Controls.Symbology)) {
+                this.legendItem.displaySymbology = !this.legendItem.displaySymbology;
+            }
+        },
 
-    /**
-     * Toggles data table panel to open/close for the LegendItem.
-     */
-    toggleGrid() {
-        if (this.legendItem._controlAvailable(Controls.Datatable)) {
-            this.$iApi.event.emit(GlobalEvents.GRID_TOGGLE, this.legendItem.layerUID);
-        }
-    }
+        /**
+         * Toggles data table panel to open/close for the LegendItem.
+         */
+        toggleGrid() {
+            if (this.legendItem._controlAvailable(Controls.Datatable)) {
+                this.$iApi.event.emit(GlobalEvents.GRID_TOGGLE, this.legendItem.layerUID);
+            }
+        },
 
-    /**
-     * Toggles settings panel to open/close type for the LegendItem.
-     */
-    toggleSettings() {
-        if (this.legendItem._controlAvailable(Controls.Settings)) {
-            this.$iApi.event.emit(GlobalEvents.SETTINGS_TOGGLE, this.legendItem.layerUID);
-        }
-    }
+        /**
+         * Toggles settings panel to open/close type for the LegendItem.
+         */
+        toggleSettings() {
+            if (this.legendItem._controlAvailable(Controls.Settings)) {
+                this.$iApi.event.emit(GlobalEvents.SETTINGS_TOGGLE, this.legendItem.layerUID);
+            }
+        },
 
-    /**
-     * Toggles metadata panel to open/close for the LegendItem.
-     */
-    toggleMetadata() {
-        if (this.legendItem._controlAvailable(Controls.Metadata)) {
-            // TODO: toggle metadata panel through API/store call
-            this.$iApi.event.emit('metadata/open', {
-                type: 'html',
-                layer: 'Sample Layer Name',
-                url: 'https://ryan-coulson.com/RAMPMetadataDemo.html'
-            });
-        }
-    }
+        /**
+         * Toggles metadata panel to open/close for the LegendItem.
+         */
+        toggleMetadata() {
+            if (this.legendItem._controlAvailable(Controls.Metadata)) {
+                // TODO: toggle metadata panel through API/store call
+                // this.$iApi.event.emit('metadata/open', {
+                //     type: 'html',
+                //     layer: 'Sample Layer Name',
+                //     url:
+                //         'https://ryan-coulson.com/RAMPMetadataDemo.html'
```

Comment:
uncomment this for metadata panel to work (probably just a small rebase error)

Reply:
oops, donethanks 👍